api/envoy/extensions/http/header_formatters/preserve_case/v3/preserve_case.proto

```diff
@@ -17,7 +17,21 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // See the :ref:`header casing <config_http_conn_man_header_casing>` configuration guide for more
 // information.
 message PreserveCaseFormatterConfig {
+  enum FormatterTypeOnUnknownHeaders {
+    // Use LowerCase on unknown headers.
+    DEFAULT = 0;
+
+    // Use ProperCaseHeaderKeyFormatter on unknown headers that upper cases the first character
+    // in each word: The first character as well as any alpha character following a special
+    // character is upper cased.
+    PROPER_CASE = 1;
+  }
+
   // Allows forwarding reason phrase text.
   // This is off by default, and a standard reason phrase is used for a corresponding HTTP response code.
   bool forward_reason_phrase = 1;
+
+  // Type of formatter to use on unknown headers which are added by envoy.
+  // The default type is DEFAULT, use LowerCase on unknown headers.
+  FormatterTypeOnUnknownHeaders formatter_type_on_unknown_headers = 2;
```

Comment:
```suggestion
  FormatterTypeOnUnknownHeaders formatter_type_on_envoy_headers = 2;
```

Reply:
Generally I would replace unknown with Envoy everywhere in this file for clarity.

Reply:
done